apps/academy/src/components/Header.tsx

```diff
@@ -36,42 +41,44 @@ const PageHeader: FunctionComponent = () => {
   const { pathname } = router;
 
   return (
```

Comment:
there is like a line between the header and the rest of the page. happening on fundamentals and tracks too.

![image](https://github.com/user-attachments/assets/6226fa3f-e5f4-476c-a2d3-a03fd9095657)

![image](https://github.com/user-attachments/assets/ccfcf91e-d13e-45df-b30f-06076667ff64)

can you check it?

Reply:
yeah, about this. 
the image was placed in the background with an overlay covering the entire site due to how the code was structured. here's the CSS file for reference.
```
@import "ui/styles.css";

/* Apply the background to the entire page */
body {
  margin: 0; /* Remove default margin */
  height: 100vh;
  position: relative; /* Ensure child elements are positioned relative to body */
}

body::after {
  content: ""; /* Required to generate a pseudo-element */
  position: absolute; /* Position overlay absolutely within body */
  top: 0;
  left: 0;
  width: 100%;
  height: 2387px;
  background: url("/home_bg.png") no-repeat center center;
  background-size: cover; /* Make the background image cover the entire page */
  opacity: 0.4; /* Adjust this value to change the opacity of the background image */
  z-index: -1; /* Ensure the overlay is behind other content */
}
```
<img width="1677" alt="Screenshot 2024-09-14 at 12 45 02 PM" src="https://github.com/user-attachments/assets/4bed2f5b-0da1-41f4-aca7-b6088e8731c5">


so I noticed there's no background on the tracks and fundamentals pages, just a background color in the div. i can either raise another PR to fix both pages or include the fix in this PR 
what do you think?

Reply:
> I noticed there's no background on the tracks and fundamentals pages, just a background color in the div. 

Yeah, those pages don't have background. Figma screenshot for reference:
![image](https://github.com/user-attachments/assets/c78086b4-953a-4820-b454-401fe0b587be)



> i can either raise another PR to fix both pages or include the fix in this PR what do you think?

I believe it belongs to this PR but honestly feel really free to manage it as your best convenience mate :muscle: 
As far I saw we shouldn't put the new home_bg image on the body or near that.

LFG :rocket:

Reply:
The issue is fixed.
<img width="1688" alt="Screenshot 2024-09-16 at 12 40 20 PM" src="https://github.com/user-attachments/assets/3d3a0f32-b52d-401e-944f-2078588d643d">
<img width="1691" alt="Screenshot 2024-09-16 at 12 40 06 PM" src="https://github.com/user-attachments/assets/b6f6c704-630e-4fa4-8509-92c632422788">